microsoft-365/admin/setup/download-software-licenses-csp.md

```diff
@@ -34,7 +34,7 @@ You must be a Global admin to do the steps in this article. For more information
 ## Download software and product license keys
 
 1. In the Microsoft 365 admin center, go to the **Billing** > <a href="https://go.microsoft.com/fwlink/p/?linkid=842054" target="_blank">Your products</a> page.
-2. On the **Products** tab, in the **Software** section, select the software that you want to download.
+2. On the **Products** tab, in the **Software** section, click on the product to navigate to the subscription details page.
```

Comment:
```suggestion
2. On the **Products** tab, in the **Software** section, select the product to navigate to the subscription details page.
```

Reply:
@msbemba Please make the change suggest above for line 37.